cpp/ql/test/library-tests/dataflow/dataflow-tests/has-parameter-flow-out.ql

```diff
@@ -0,0 +1,53 @@
+import TestUtilities.InlineExpectationsTest
+import cpp
+
+module AstTest {
+  private import semmle.code.cpp.dataflow.old.DataFlow::DataFlow
+  private import semmle.code.cpp.dataflow.old.internal.DataFlowPrivate
```

Comment:
Since https://github.com/github/codeql/pull/12148 was merged:
```suggestion
  private import semmle.code.cpp.dataflow.DataFlow::DataFlow
  private import semmle.code.cpp.dataflow.internal.DataFlowPrivate
```

Reply:
Thanks 🙇!

Reply:
Not sure if the will work directly on this PR, might need to merge in the latest version of the feature branch.

Reply:
Yeah, I'll merge in the latest version if CI is unhappy.